casepro/contacts/models.py

```diff
@@ -190,7 +190,7 @@ def get_or_create_from_urn(cls, org, urn, name=None):
         """
         contact = cls.objects.filter(urns__contains=[urn]).first()
         if not contact:
-            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=True)
+            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=False)
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/109/commits/bfd1f847b09bb2a9d5be6ef2c4c1bc1fd92a7800